src/orderbook.js

```diff
@@ -106,7 +140,16 @@ function OrderBook(remote,
       switch (action) {
         case 'add':
           if (++self._listeners === 1) {
+            self._shouldSubscribe = true;
             self.subscribe();
+
+            self._remote.on('transaction', updateFundedAmountsWrapper);
+            self._remote.once('disconnect', onDisconnect);
+
+            if (self._isAutobridgeable) {
```

Comment:
I didn't found the pattern of naming private functions with underscore here, so I left it as is. 
But yes, it would be better to rename all private function to start with underscore.